forc-plugins/forc-doc/src/doc.rs

```diff
@@ -215,6 +218,27 @@ impl ModuleInfo {
         }
         iter.map(|s| s.as_str()).collect::<Vec<&str>>().join("::")
     }
+    /// Renders the [ModuleInfo] into a [CallPath] with anchors. We return this as a `Result<Vec<String>>`
+    /// since the `box_html!` macro returns a closure and no two closures are considered the same type.
+    pub(crate) fn get_anchors(&self) -> Result<Vec<String>> {
+        let mut count = self.depth();
+        let mut rendered_module_anchors = Vec::new();
```

Comment:
Could this be changed to create the vector with capacity?
It appears to be based on the number of prefixes so if you get that length then you can allocate once instead of pushing and reallocating.

Reply:
good call, thanks!